### PR TITLE
Fix typo in usage

### PR DESCRIPTION
**Pull request by zulcss**
Should be '-r' rather than '-n'.

Signed-off-by: Charles Short <chucks@redhat.com>

### CODE REVIEW

File: agent/util-scripts/pbench-clear-tools [hunk 1]
```diff
@@ -27,7 +27,7 @@ if [ $? -ne 0 ]; then
 	printf -- "\t-g str --group=str, str = the group from which tools should be removed\n"
 	printf -- "\t                          (the default group is 'default')\n"
 	printf -- "\t-n str --name=str, str = a specific tool to be removed.  If no tool is specified, all tools in the group are removed\n"
-        printf -- "\t-n str --remote=str, str = a specific remote on which tools needs to be cleared.  If no remote is specified, all the tools are removed\n"
+        printf -- "\t-r str --remote=str, str = a specific remote on which tools needs to be cleared.  If no remote is specified, all the tools are removed\n"
```
Comment (portante) [inline agent/util-scripts/pbench-clear-tools]: @zulcss, can you fix the tab vs whitespace here?

File: agent/util-scripts/pbench-clear-tools [hunk 2]
```diff
@@ -24,10 +24,10 @@ if [ $? -ne 0 ]; then
 	printf "\n"
 	printf "$script_name: you specified an invalid option\n\n"
 	printf "The following are required:\n\n"
-	printf -- "\t-g str --group=str, str = the group from which tools should be removed\n"
+	printf -- "\t   -g str --group=str, str = the group from which tools should be removed\n"
 	printf -- "\t                          (the default group is 'default')\n"
-	printf -- "\t-n str --name=str, str = a specific tool to be removed.  If no tool is specified, all tools in the group are removed\n"
-        printf -- "\t-n str --remote=str, str = a specific remote on which tools needs to be cleared.  If no remote is specified, all the tools are removed\n"
+	printf -- "\t   -n str --name=str, str = a specific tool to be removed.  If no tool is specified, all tools in the group are removed\n"
+        printf -- "\t   -r str --remote=str, str = a specific remote on which tools needs to be cleared.  If no remote is specified, all the tools are removed\n"
```
Comment (portante) [inline agent/util-scripts/pbench-clear-tools]: It seems we want to leave the internal formatting of the `printf` statements intact, but fix the whitespace before the `printf` to be tabs, not spaces.

Comment (portante) [reply agent/util-scripts/pbench-clear-tools]: Consider merging https://github.com/zulcss/pbench/pull/1.

**Comment by zulcss** [comment]
done

**Pull request merged**